domains/auth/Auth.API/API/Controllers/AuthController.cs

```diff
@@ -30,7 +34,22 @@ public AuthController(ILogger<AuthController> logger, IOidcProviders oidcProvide
             ReturnUrl = returnUrl
         };
 
-        return oidcProviders.CreateAuthorizationUri(state);
+        return _oidcProviders.CreateAuthorizationUri(state);
     }
 
+    [HttpPost]
+    [Route("/auth/logout")]
+    public ActionResult<LogoutResponse> Logout()
+    {
+        var token = HttpContext.Request.Headers[_authOptions.CookieName].FirstOrDefault()?.Split(" ").Last();
+
+        if (token != null)
+        {
+            //delete token from database
+            //oidcProviders.Logout(token);
```

Comment:
Why is this commented out?

Reply:
These are outcommented due to their functions havent been implemted yet.